client/asset/dcr/dcr.go

```diff
@@ -1316,8 +1416,19 @@ func (dcr *ExchangeWallet) split(value uint64, lots uint64, coins asset.Coins, i
 		return coins, false, nil
 	}
 
-	// Use an internal address for the sized output.
-	addr, err := dcr.wallet.InternalAddress(dcr.ctx, dcr.acctName)
+	// Generate an address to receive the sized output. If mixing is
+	// enabled on the wallet, generate the address from the external
+	// branch of the trading account. The external branch is used so
+	// that if this split output isn't spent, it won't be transferred
+	// to the unmixed account for re-mixing. Instead, it'll simply be
+	// unlocked in the trading account and can thus be used to fund
+	// future orders.
```

Comment:
My VS code says 4. I'll like to see how this looks on your end if you edit the wrapping. Can you share as a comment?

Reply:
Before:

![image](https://user-images.githubusercontent.com/9373513/168097207-da3880e3-7137-4d7b-a0b6-e3ef353f360f.png)

Notice the cursor at col 70 of the longest line there, also "Tab Size 4".

Alt+Q to wrap after 80 cols:

![image](https://user-images.githubusercontent.com/9373513/168097437-83b387c2-2102-4375-9c7a-ff0b593c4555.png)

Note that while it says the cursor is at 80, it would also be good if it say 81 because when at an empty line it says 1.

I think while your editor is configured for 4-space tabs, your wrapping extension is not.